src/core/settings.hpp

```diff
@@ -336,6 +336,10 @@ namespace big
 			bool custom_weapon_stop    = true;
 			std::string persist_outfit = "";
 			bool persist_outfits_mis   = false;
+			bool nav                   = false;
```

Comment:
You could create a nav struct, something like:
```cpp
struct nav
{
	enabled = false;
	voice      = false;
	text        = false;
	NLOHMANN_DEFINE_TYPE_INTRUSIVE(nav, enabled, voice, text)
} nav {};
```

Reply:
Good Idea, I will implement this